### DISCUSSION TITLE
why my bing mode  doesn't work

### DISCUSSION THREAD

**Discussion by Maddaxman** [waylaidwanderer/node-chatgpt-api#389] [Help] [answered]
Why can I successfully generate content on the Bing mode through the API of the node-chatgpt-api project using chatgptbox, but cannot generate content on PandoraAI even though I have modified the API_BASE_URL address and there is no response after sending a message?

**Comment by waylaidwanderer** [comment, marked as answer]
Pull the latest version of PandoraAI and try again. The latest Nuxt update changed how the env var was being set, so I had to update the frontend code to account for this, which should work now.

**Reply by Maddaxman** [reply]
> Pull the latest version of PandoraAI and try again. The latest Nuxt update changed how the env var was being set, so I had to update the frontend code to account for this, which should work now.

its work,thx
